ui/src/environments/environment-type.ts

```diff
@@ -18,6 +18,9 @@ export interface Environment {
   // The Google Analytics account ID for logging actions and page views.
   // Example value: 'UA-112406425-3'
   gaId: string;
+  // The Google Analytics custom dimension ID for sending User Agent
+  // info, allowing us to filter out Pingdom, Appscan, etc
+  gaDimension: string;
```

Comment:
Naming nitpick: since we may have *other* dimension in the future, I'd go with something like

"gaUserAgentDimension"

I'd also update the comment to include an explicit example, e.g. "dimension1". Just so it's extremely clear to everyone what the format should be :)